DancingGoat/Scripts/PreviewMode/PreviewMode.js

```diff
@@ -0,0 +1,107 @@
+﻿const previewModeToggleClassname = 'preview-mode-switch';
+const previewModeCookieName = 'IsPreviewMode';
+const previewApiKeyCookieName = 'PreviewApiKey';
+const enterApiKeyFirstMessage = 'Enter your Preview API key';
+const enterApiKeyNextMessage = 'Enter your Delivery Preview API key -- you can find it in Kentico Cloud under Project settings -> API keys';
```

Comment:
Maybe rather Kentico Cloud App?

Reply:
This copy is provided by Aaron, I'll keep it as is.